cmd/daemon/serve.go

```diff
@@ -147,7 +155,11 @@ func ServeAdmin(r driver.Registry, wg *sync.WaitGroup, cmd *cobra.Command, args
 	}
 
 	n.UseHandler(router)
-	server := graceful.WithDefaults(&http.Server{Handler: context.ClearHandler(n)})
+	certs := c.GetTSLCertificatesForAdmin()
+	server := graceful.WithDefaults(&http.Server{
+		Handler:   context.ClearHandler(n),
+		TLSConfig: &tls.Config{Certificates: certs, MinVersion: tls.VersionTLS12},
```

Comment:
What's the best practice around `MinVersion: tls.VersionTLS12`? Is there some documentation regarding this you could link to?

Reply:
I added it because one of the linters failed here https://app.circleci.com/pipelines/github/ory/kratos/4046/workflows/0883c837-4188-4a6b-90e6-a48669fa4deb/jobs/22947/parallel-runs/0/steps/0-121

And I found a solution in this discussion - https://github.com/go-redis/redis/issues/1553. I don't have a lot of context of whether it is a strict requirement or not, but it fixed the problem.

Reply:
Ah nice, SGTM